### ISSUE TITLE
Please update register-service-worker dependency

### ISSUE THREAD

**Issue by 185driver** [vuejs/vue-cli#2964]
### What problem does this feature solve?
Version 1.5.2 of the `register-service-worker` package has been released, but the version used by @vue/cli-plugin-pwa remains at '1.0.0'.

Thank you for all your work. It's much appreciated.

### What does the proposed API look like?
[register-service-worker](https://github.com/yyx990803/register-service-worker)

**Comment by LinusBorg** [comment]
The rquired version is `^1.0.0`, which means it will install the latest minor version. So you will get 1.5.2, and as long as the code that we use is compatible with 1.0.0, keeping the dependency at `^1.0.0` seems fine to me.

**Comment by 185driver** [comment]
Okay, np, you would know best. 

My own experience is that I recently created a new app using vue cli 3 and added the pwa plugin as part of the creation process. An older `"register-service-worker": "^1.0.0"` was created for me in my project root's package.json `"dependencies"` section, which I manually updated to '^1.5.2'.  But it my later observation that the pwa plugin itself used the same '^1.0.0' version, which is what triggered this submitted issue. It sure seems like only the old version is being used.

On a slightly related note (and this is not the place to request an update, I know) a more complete `registerServiceWorker.js` script is provided in the [Usage section](https://github.com/yyx990803/register-service-worker#usage) for that plugin, but that too is not currently being deployed with vue's pwa plugin on project creation.
